### ISSUE TITLE
How to change log level of CocoaMQTT?

### ISSUE THREAD

**Issue by SanyasiraoM** [emqtt/CocoaMQTT#162]
How to change the log level of CocoaMQTT?

I tried below but unable to compile
CocoaMQTTLogger().minLevel = .warning
CocoaMQTTLogger.logger.minLevel = .warning

**Comment by SanyasiraoM** [comment]
@CrazyWisdom is a new version published or will it be resolved with the same 1.0.19?

**Comment by alexyu1989** [comment]
@SanyasiraoM @sotpt @CrazyWisdom 
use:` mqtt.logLevel = .error`
Making logger singleton public is not a good idea.
```
// log
    open var logLevel: CocoaMQTTLoggerLevel {
        get {
            return CocoaMQTTLogger.logger.minLevel
        }
        set {
            CocoaMQTTLogger.logger.minLevel = newValue
        }
    }
